lib/utils/constants.dart

```diff
@@ -8,18 +8,51 @@ class AppConstants {
     sortingOptions.optionNotifier.value = sortOption;
   }
 
+  // static final colorLib ={
+  //   'cyan'
+  // }
+
   static final colorsByMood = {
     'anxious': AppColors.cyan,
     'sleepless': AppColors.red,
     'energetic': AppColors.orange,
     'other': AppColors.green
   };
 
+  static final gradsByMood = {
+    'anxious': AppGrads.purple,
+    'sleepless': AppGrads.sunset,
+    'energetic': AppGrads.orange,
+    'other': AppGrads.green
+  };
+
   static Color getColorByMood(String mood) {
     var color = colorsByMood[mood.toLowerCase()];
     return color ?? const Color(0x0);
   }
 
+  /* I had some issues when I tried to build the system same to the getColorsByMood
+  function, so I just made it a switch statement for now.
```

Comment:
Please create an issue to fix this in the future.

Reply:
Will do! Do you imagine that the way it's currently set up will cause a problem later on down the line? Or is it just better to have getColorByMood and get GradByMood to be similarly constructed?

Reply:
getGradByMood might be a better idea. We might need the fixed colors in the future too.